### PR TITLE
EPH Tracking - Heat Related Illness

### PR DESCRIPTION
**Pull request by Abilityguy**
Artifacts used for the Heat Related illness import, includes a readme and tests.

### CODE REVIEW

File: scripts/us_cdc/heat_related_illness/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Importing EPH Heat and Heat-related Illness data
+
+This directory imports [Heat and Heat-related Illness](https://ephtracking.cdc.gov/qrlist/35) from EPH Tracking into Data Commons. It includes data at a state level.
+
+## Cleaning source data
+To clean the source data, run:
```
Comment (ajaits) [inline scripts/us_cdc/heat_related_illness/README.md:6]: Pls indicate if the data needs to be downloaded manually when new files are available.

File: scripts/us_cdc/heat_related_illness/preprocess.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+import json
+import csv
+import copy
+
+from absl import app
+from absl import flags
+
+# Allows the following module imports to work when running as a script
+_SCRIPT_PATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(_SCRIPT_PATH, '../../../util/'))
```
Comment (ajaits) [inline scripts/us_cdc/heat_related_illness/preprocess.py:27]: sys.path.append(os.path.join(_SCRIPT_PATH, 'os.path.join('..', os.path.join('..', os.path.join('..', 'util'))

File: scripts/us_cdc/heat_related_illness/preprocess_test.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            ])
+
+            with open(csv_path, 'r', encoding='utf-8') as f_result:
+                test_result = f_result.read()
+                expected_csv_path = os.path.join(_SCRIPT_PATH, 'testdata',
+                                                 'expected.csv')
+                with open(expected_csv_path, 'r',
+                          encoding='utf-8') as f_expected:
+                    expected_result = f_expected.read()
+                self.assertEqual(test_result, expected_result)
```
Comment (ajaits) [inline scripts/us_cdc/heat_related_illness/preprocess_test.py:45]: Can we validate output.mcf as well?

File: scripts/us_cdc/heat_related_illness/README.md (hunk 1 above)
Comment (Abilityguy) [reply scripts/us_cdc/heat_related_illness/README.md:7]: Done.

File: scripts/us_cdc/heat_related_illness/preprocess.py (hunk 2 above)
Comment (Abilityguy) [reply scripts/us_cdc/heat_related_illness/preprocess.py]: Done.

File: scripts/us_cdc/heat_related_illness/preprocess.py [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+    Processes the data file using csv_reader and writes processed data to
+    csv_writer.
+    """
+    template_statvar = {**_CONFIG["filename"][file_name]}
+    data_col = 'Data Value'
+
+    statvars = []
+    for row in csv_reader:
+        quantity = row[data_col]
+        if quantity == 'Suppressed':  # SKip suppressed data
```
[the remark this answers is not in the document]
Comment (Abilityguy) [reply scripts/us_cdc/heat_related_illness/preprocess.py]: Done.

File: scripts/us_cdc/heat_related_illness/preprocess_test.py (hunk 3 above)
Comment (Abilityguy) [reply scripts/us_cdc/heat_related_illness/preprocess_test.py:55]: Done, have included a check to the .mcf file in the test.

**Review by ajaits — APPROVED** [review]

**Pull request merged**